src/theme/variables/airbitz.js

```diff
@@ -76,13 +76,14 @@ export const THEME = {
     ACCENT_RED: '#E85466',
     ACCENT_MINT: '#66EDA8',
 
-    GENERIC_BLACK: '#000000',
-    GENERIC_RED: '#ff0000',
-    GENERIC_ORANGE: '#ffa500',
-    GENERIC_GREEN: '#008000',
-    GENERIC_PURPLE: '#800080',
-    GENERIC_YELLOW: '#ffff00',
-    GENERIC_GREY: '#808080',
+    GENERIC_BLACK: 'black',
+    GENERIC_RED: 'red',
+    GENERIC_ORANGE: 'orange',
+    GENERIC_GREEN: 'green',
+    GENERIC_PURPLE: 'purple',
+    GENERIC_YELLOW: 'yellow',
+    GENERIC_GREY: 'grey',
+    GENERIC_WHITE: 'white',
```

Comment:
What is the difference between `GENERIC_WHITE` and just `WHITE`? Why would anybody chose one over the other?